### PR TITLE
SEQNG-750 Introduced class ExecutionQueueView for web server.

### PR DESCRIPTION
**Pull request by jluhrs**
Also, redefined `ClientId` and `QueueId` to avoid ambiguities.

### CODE REVIEW

File: modules/seqexec/model/src/test/scala/seqexec/model/SeqexecModelArbitraries.scala [hunk 1]
```diff
@@ -274,15 +282,16 @@ trait SeqexecModelArbitraries extends ArbObservation {
...
-    Cogen[
-      (StepId,
-       Map[SystemName, Map[String, String]],
-       StepState,
-       Boolean,
-       Boolean,
-       Option[dhs.ImageFileId],
-       List[(Resource, ActionStatus)],
-       ActionStatus)].contramap(
+    Cogen[(
```
Comment (jluhrs) [inline modules/seqexec/model/src/test/scala/seqexec/model/SeqexecModelArbitraries.scala]: Scalastyle was complaining about this in Idea.

File: modules/seqexec/model/src/main/scala/seqexec/model/package.scala [hunk 2]
```diff
@@ -4,9 +4,15 @@
 package seqexec
 
 import cats._
+import cats.implicits._
 import java.util.UUID
 import seqexec.model.enum._
 
+package model {
+  final case class QueueId(self: UUID) extends AnyVal
```
Comment (cquiroz) [inline modules/seqexec/model/src/main/scala/seqexec/model/package.scala]: 👍

File: modules/seqexec/model/src/test/scala/seqexec/model/SeqexecModelArbitraries.scala [hunk 3]
```diff
@@ -43,7 +45,14 @@ trait SeqexecModelArbitraries extends ArbObservation {
     } yield SequencesQueue(Map.empty, c, o, Map.empty, b)
   }
 
-  implicit val clientIdArb: Arbitrary[ClientID] = Arbitrary(Gen.uuid)
+  implicit val arbitraryUUID: Arbitrary[UUID] = Arbitrary(Gen.uuid)
+
+  implicit val cogenUUID: Cogen[UUID] =
+    Cogen[(Long, Long)].contramap(u => (u.getMostSignificantBits, u.getLeastSignificantBits))
```
Comment (cquiroz) [inline modules/seqexec/model/src/test/scala/seqexec/model/SeqexecModelArbitraries.scala]: Nice

File: modules/seqexec/web/shared/src/main/scala/seqexec/web/model/boopickle/ModelBooPicklers.scala [hunk 4]
```diff
@@ -114,7 +115,7 @@ trait ModelBooPicklers extends GemModelBooPicklers {
     .addConcreteType[BatchCommandState.Run]
     .addConcreteType[BatchCommandState.Stop.type]
 
-  implicit val executionQueuePickler = generatePickler[ExecutionQueue]
+  implicit val executionQueuePickler = generatePickler[ExecutionQueueView]
```
Comment (cquiroz) [inline modules/seqexec/web/shared/src/main/scala/seqexec/web/model/boopickle/ModelBooPicklers.scala]: Oh I missed this

Comment (cquiroz) [reply modules/seqexec/web/shared/src/main/scala/seqexec/web/model/boopickle/ModelBooPicklers.scala]: we still need a explicit boopickler for all fields, like `BatchExecState`

Comment (jluhrs) [reply modules/seqexec/web/shared/src/main/scala/seqexec/web/model/boopickle/ModelBooPicklers.scala]: Done.

**Pull request merged**